singlestore/assets/dashboards/overview.json

```diff
@@ -1,6 +1,6 @@
 {
     "author_name": "Datadog",
-    "description": "## SingleStore\n\nThis dashboard provides a high-level view of your SingleStore metrics and can help you troubleshoot performance issues. For further reading on SingleStore:\n- [Datadog's SingleStore integration docs](https://docs.datadoghq.com/integrations/singlestore/)\n- [SingleStore docs](https://www.singlestore.com/)\n",
+    "description": "## SingleStore\n\nThis dashboard provides a high-level view of your SingleStore metrics and can help you troubleshoot performance issues. For further reading on SingleStore:\n- [Datadog's SingleStore integration docs](https://github.com/DataDog/integrations-core/tree/master/singlestore)\n- [SingleStore docs](https://www.singlestore.com/)",
```

Comment:
Why is the Datadog SingleStore docs changing to the Github repo?

Reply:
hey @yzhan289  thanks for reviewing- those were the changes in the dashboard in the Jira card, so I included them. But I can revert it if thats better- i was also a bit unsure why that was changed.

Reply:
Let's keep it as docs.datadoghq.com for consistency.